.tool-versions

```diff
@@ -1,3 +0,0 @@
-ruby 3.2.2
```

Comment:
only need go now, and README.md now tells devs to install the version specified in our Github action, so will always be consistent